assets/base.css

```diff
@@ -2116,6 +2116,46 @@ product-info .loading-overlay:not(.hidden) ~ *,
   border-bottom: 0.1rem solid rgba(var(--color-foreground), 0.08);
 }
 
+@media screen and (min-width: 990px) {
+  .utility-bar--bottom-border-social-only {
+    border-bottom: 0.1rem solid rgba(var(--color-foreground), 0.08);
+  }
+}
+
+.utility-bar__grid {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.utility-bar__grid .list-social {
+  justify-content: flex-start;
+  align-content: center;
+  margin-left: -1.2rem;
+}
+
+@media screen and (max-width: 989px) {
+  .utility-bar .utility-bar__grid .list-social {
+    display: none;
```

Comment:
Thanks for brining it up. In my opinion we can assume it's something intuitive, however if you feel we should raise this question, I'm okay with that. 

@YoannJailin what do you think about the fact that show social-icons on the screen size up to 990px and for smaller screens we hide them because we always show icons in the drawer? Do we mention about it in the settings?

Reply:
@YoannJailin @melissaperreault could you pls step in and help us decide? I can easily add info text, I just need UX approval for this :)

Reply:
@eugenekasimov i agree we should be explicit that some features only shows up on mobile. Let me take a look on that this afternoon.

Reply:
@sofiamatulis we decided to change the label to `Show icons on desktop`. Thanks for brining up this question. 

fixed here [dc6dd49](https://github.com/Shopify/dawn/pull/2497/commits/dc6dd49362957c3995e329644d201a563a60d193)